distribution/server-dist/assembly.xml

```diff
@@ -118,6 +118,13 @@
                 <include>licenses-${product.slot}/**</include>
             </includes>
         </fileSet>
+        <fileSet>
+            <directory>src/main/docs</directory>
+            <outputDirectory>docs</outputDirectory>
+            <includes>
+                <include>**</include>
+            </includes>
+        </fileSet>
```

Comment:
Can you please explain why we need this change? Thank you

Reply:
yes, that's correct